scripts/auto_sshfs_mount.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Dec  3 07:16:00 2019
+
+@author: danjampro
+
+Run this on the pis (not the main computer).
+"""
+import os, subprocess
+from huntsman.utils import load_config
+
+#==============================================================================
+
+def Mount(local, remote, ServerAliveInterval=20, ServerAliveCountMax=3):
+    '''
+    Mount remote on local.
+    '''
+    if not os.path.isdir(local):
+        os.mkdir(local)
+        
+    options = f'ServerAliveInterval={ServerAliveInterval},' + \
+              f'ServerAliveCountMax={ServerAliveCountMax}'
+    options = ['sshfs', remote, local, '-o', options]
+    try:
+        subprocess.run(options, shell=False, check=True)
+    except Exception as e:
+        print(f'Failed to mount {remote} on {local}.')
+        raise(e)
+    
+    
+def AttemptUnmount(local):
+    '''
+    Unmount remote from local.
+    '''
+    if os.path.isdir(local):
+        options = ['fusermount', '-u', local]
+        try:
+            subprocess.run(options, shell=False, check=True)
+        except:
+            pass
+        
+#==============================================================================
+#==============================================================================
+
+if __name__ == '__main__':
+    
+    #Specify the mount point as ~/Huntsmans-Pro (for huntsman user)
+    local = '/home/huntsman/Huntsmans-Pro'
+    
+    #Specify the remote directory
+    dirs = load_config()['directories']
+    remote = f"huntsman@192.168.80.101:{dirs['base']}"
```

Comment:
I think this doesn't matter because it isnt our external ip address(?)... but we have had an issue before with an ip going into the repos commit history/not being easily able to remove it.

Reply:
Good point. I've updated it so that the IP is loaded from the local config file.